cmd/kops-controller/controllers/node_controller.go

```diff
@@ -119,6 +119,15 @@ func (r *NodeReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, fmt.Errorf("unable to build config for node %s: %v", node.Name, err)
 	}
 
+	spot, err := r.getInstanceLifecycle(ctx, node)
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("unable to get instance lifecycle %s: %v", node.Name, err)
+	}
+
+	if spot {
+		labels["node-role.kubernetes.io/spot-worker"] = "spot"
```

Comment:
The documented value seems to be "true":
https://github.com/pusher/k8s-spot-termination-handler/blob/master/README.md#requirements
```suggestion
		labels["node-role.kubernetes.io/spot-worker"] = "true"
```